modular_skyrat/modules/ashwalkers/code/buildings/antfarm.dm

```diff
@@ -0,0 +1,64 @@
+/obj/structure/antfarm
+	name = "ant farm"
+	desc = "Though it may look natural, this was not made by ants."
+	icon = 'modular_skyrat/modules/ashwalkers/icons/structures.dmi'
+	icon_state = "anthill"
+	density = TRUE
+	anchored = TRUE
+	/// If the farm is occupied by ants
+	var/has_ants = FALSE
+	/// the chance for the farm to get ants
+	var/ant_chance = 0
+	/// the list of ore-y stuff that ants can drag up from deep within their nest
+	var/list/ore_list = list(
+		/obj/item/stack/ore/iron = 20,
+		/obj/item/stack/ore/glass = 20,
+		/obj/item/stack/sheet/mineral/plasma = 14,
```

Comment:
Why?

Reply:
the ores and stuff block their way from digging down, so they throw it out of their nest.

Reply:
As in, why is this something that Ashwalkers need?

Reply:
its a slow method of gathering passive materials-- which means people can focus on other aspects of the game; also, there are a ton of ways to get materials now which are far easier than this.